### PR TITLE
Add Resend

### PR DESCRIPTION
**Pull request by bodinsamuel**
**Similarweb rank:** 174,384

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
https://resend.com/

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @bodinsamuel!
The JSON structure looks spot on, but I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/54ed682e-ece6-4d8f-bba4-94278bb2fc33) between the black version of the SVG from their site, and your icon. Looking at that screenshot, it also looks like they've not properly optimised the SVG path, and the 'arm' of the R is still a separate path. If you run an `Add` in Inkscape, that should fix that right up! Let me know if you're having any issues, and I can provide a path - but we'll need another maintainer to review if we go that route!

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @bodinsamuel?

File: icons/resend.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Resend</title><path d="M2.12 24.023V-.061h10.837c1.537 0 2.924.333 4.163.998 1.261.665 2.248 1.582 2.959 2.752.71 1.147 1.066 2.454 1.066 3.922 0 1.514-.355 2.879-1.066 4.094a7.681 7.681 0 0 1-2.925 2.822c-1.238.688-2.638 1.032-4.197 1.032H7.693v8.464H2.119zm13.348 0L9.344 13.15l5.987-.86 6.812 11.732h-6.675zM7.693 11.086h4.748c.596 0 1.112-.126 1.548-.378a2.6 2.6 0 0 0 1.032-1.136c.252-.481.379-1.032.379-1.651 0-.62-.138-1.158-.413-1.617a2.677 2.677 0 0 0-1.204-1.101c-.505-.252-1.124-.378-1.858-.378H7.693v6.261z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/resend.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Resend</title><path d="M2.023 0v24h5.553v-8.434h2.998L15.326 24h6.65l-5.372-9.258a7.652 7.652 0 0 0 3.316-3.016c.709-1.21 1.062-2.57 1.062-4.08 0-1.462-.353-2.767-1.062-3.91-.709-1.165-1.692-2.079-2.95-2.742C15.737.331 14.355 0 12.823 0Zm5.553 4.87h4.219c.731 0 1.349.125 1.851.376.526.252.925.618 1.2 1.098.274.457.412.994.412 1.611S15.132 9.12 14.88 9.6c-.229.48-.572.856-1.03 1.13-.434.252-.948.38-1.542.38H7.576Z"/></svg>
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -10924,6 +10924,11 @@
             "hex": "00CCBB",
             "source": "https://c5.rgstatic.net/m/428059296771819/images/favicon/favicon.svg"
         },
+        {
+            "title": "Resend",
+            "hex": "000000",
+            "source": "https://resend.com/brand"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:10930]:
```suggestion
            "source": "https://resend.com/brand",
            "guidelines": "https://resend.com/brand"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**